CHANGELOG.md

```diff
@@ -1,6 +1,57 @@
 Change log
 ==========
 
+1.9.0 (2016-10-20)
+-----------------
+
+**Breaking changes**
+
+- For users of Docker Toolbox / Machine on Windows, the path conversion
+  for volumes is no longer done by default. To enable it, please set the
+  `COMPOSE_CONVERT_WINDOWS_PATHS` variable.
+
+New Features
+
+- Interactive mode for `docker-compose run` and `docker-compose exec` is
+  now supported on Windows platforms.
+
+- Introduced version 2.1 of the `docker-compose.yml` specification. Using
+  this version requires to be used with Docker Engine 1.12 or above.
+    - Added support for setting volume labels and network labels in
+  `docker-compose.yml`.
+    - Added support for the `isolation` parameter in service definitions.
+    - Added support for link-local IPs in the service networks definitions.
+
+- Added support for inline defaults in variable interpolation.
```

Comment:
Could mention the syntax here, so people don't have to go hunting in the docs:

Added support for shell-style inline defaults in variable interpolation. The supported forms are `${FOO:-default}` (fall back if FOO is unset) and `${FOO-default}` (fall back if FOO is unset or empty).